enaburekhan/ror-social-scaffold
# Milestone 3: friendship-v2

**Milestone 3: Friendship-v2**
**In this project:**

- Add integration test for sign-in, sign-out, and friendship
- Add negative test scenarios
- Add button to add and remove friends
- Add user controller test
- Add post controller test
- A signed-in user should not be able to see another user's post in the timeline until that user accepts the friend request. This is working perfectly
- When a signed-in user goes to the profile of another user who is not a friend, he should not be able to view their post. This is also working perfectly.
